packages/light-client/test/config.js

```diff
@@ -0,0 +1,44 @@
+module.exports = function getConfig(env) {
```

Comment:
Is there a specific reason why we're writing tests in js and not ts?

Reply:
no idea, was following the format of existing tests when I wrote